pinot-core/src/test/resources/data/newColumnsSchema3.json

```diff
@@ -16,6 +16,11 @@
     {
       "dataType": "DOUBLE",
       "name": "newDoubleMetric"
+    },
+    {
+      "dataType": "BYTES",
+      "defaultNullValue": "00000008000000ac0000",
```

Comment:
is this hexstring of the byte array

Reply:
This is one value of hyperLogLog

Reply:
Yes, this is helix string of the byte array.